src/playwright/playwright-plugin.ts

```diff
@@ -34,14 +24,13 @@ export class PlaywrightPlugin extends BrowserPlugin<BrowserType, Parameters<Brow
 
             if (!this._browserVersion) {
                 // Launches unused browser just to get the browser version.
-
                 const inactiveBrowser = await this.library.launch(launchOptions);
                 this._browserVersion = inactiveBrowser.version();
 
-                inactiveBrowser.close().catch(noop);
+                await inactiveBrowser.close();
```

Comment:
We should not need to wait for the browser to close and we're not really interested if the close fails for some reason.

Reply:
If this fails to close 100x times then memory consumption goes up.

Reply:
That's true, but does it? Maybe we could debug log the errors instead of throwing. It's an internal operation.

Reply:
> It's an internal operation.

Yup, therefore imo this should be handled gracefully instead of fire and forget.

Reply:
I think `close`no less important than `launch()` and `version()`.

Reply:
If this turns out to be unreliable we can revert this change later.